tools/plugins/com.liferay.ide.ui/META-INF/MANIFEST.MF

```diff
@@ -30,7 +30,8 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.sapphire.ui;bundle-version="[9,10)",
  org.eclipse.sapphire.ui.swt.xml.editor;bundle-version="[9,10)",
  org.eclipse.sapphire.modeling;bundle-version="[9,10)",
- com.liferay.ide.sdk.core
+ com.liferay.ide.sdk.core,
+ org.eclipse.epp.logging.aeri.ide
```

Comment:
Terry, We need to ensure that the feature that contains this plugin is included, and if it isn't included it wont fail.  can you make this dependency optional?  Thanks!